spec/views/external_users/claims/case_details/_summary.html.haml_spec.rb

```diff
@@ -138,8 +138,7 @@
     let(:claim) { present(build(:advocate_hardship_claim, :draft)) }
 
     before do
-      allow(claim).to receive(:case_stage).and_return(build(:case_stage, description: 'Case stage 101'))
```

Comment:
Hi, everything looks good to me. I think it removed the TODO comment here ... is that ok?

Reply:
Thanks. I have put the TODO comment back.